src/api/app/views/webui2/webui/search/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,9 @@
+= render partial: 'webui/main/breadcrumb_items'
+
+- case action_name
+- when 'index'
+  %li.breadcrumb-item.active{ 'aria-current' => 'page' }
```

Comment:
This line, that is the same as line 8, can be moved before the `case` statement. But this can be done later on.

Reply:
Good point, I did it now.